OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -1,42 +1,51 @@
+IMPORTANT: You have to download the `Oracle WebLogic Server Proxy Plugin`, version 12.2.1.3.0 or 12.2.1.4.0 package, (see the `.download` file) and place it in this directory.
 
-Run the `buildDockerImage.sh` script.
+Run the `buildDockerImage.sh` script with version option `-v`.
 
-        $ sh buildDockerImage.sh
+        $ sh buildDockerImage.sh -v <Verion of the image to be built>
+For Example to built the image for version 12.2.1.4.0 
+        $ sh buildDockerImage.sh -v 12.2.1.4.0
 
 ## Run the Apache HTTP Server in a Container
 
-Run an Apache container to access an Administration Server, or a Managed Server, in a non-clustered environment that is running on `<host>` and listening to `<port>`.
-
-        $ docker run -d -e WEBLOGIC_HOST=<host> -e WEBLOGIC_PORT=<port> -p 80:80 oracle/apache:12.2.1.3
+Run an Apache container on supported versions  to access an Administration Server, or a Managed Server, in a non-clustered environment that is running on `<host>` and listening to `<port>`.
 
+        $ docker run -d -e WEBLOGIC_HOST=<host> -e WEBLOGIC_PORT=<port> -p 80:80 oracle/apache:<version>
+Similarly, Apache container in NonPriviledgedPort (8080) 		
+        $ docker run -d -e WEBLOGIC_HOST=<host> -e WEBLOGIC_PORT=<port> -e NonPriviledgedPorts=true -p 8080:8080 oracle/apache:<version>
+		
 Run an Apache image to proxy and load balance to a list of Managed Servers in a cluster.
 
         Use a list of hosts and ports.
 
-        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port -p 80:80 oracle/apache:12.2.1.3
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port -p 80:80 oracle/apache:<version>
 
-        Or use a cluster URL if it is available
+Similarly, Apache container in NonPriviledgedPort (8080) 		
 
-        $ docker run -d -e WEBLOGIC_CLUSTER=<cluster-url> -p 80:80 oracle/apache:12.2.1.3
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port -e NonPriviledgedPorts=true -p 8080:8080 oracle/apache:<version>
+        Or use a cluster URL if it is available
 
+        $ docker run -d -e WEBLOGIC_CLUSTER=<cluster-url> -p 80:80 oracle/apache:<version>
+Similarly, Apache container in NonPriviledgedPort (8080) 		
+        $ docker run -d -e WEBLOGIC_CLUSTER=<cluster-url> -e NonPriviledgedPorts=true -p 8080:8080 oracle/apache:<version>
+		
 The values of `WEBLOGIC_CLUSTER` must be valid and correspond to existing containers running WebLogic Servers.
 
 ### Administration Server Only Example
 
 First, make sure that you have the WebLogic Server 12.2.1.3 install image. Pull the WebLogic install image from the DockerStore, `store/oracle/weblogic:12.2.1.3`, or build your own image, `oracle/weblogic:12.2.1.3-developer`, at [https://github.com/oracle/docker-images/tree/master/OracleWebLogic/dockerfiles/12.2.1.3](https://github.com/oracle/docker-images/tree/master/OracleWebLogic/dockerfiles/12.2.1.3).
```

Comment:
WebLogic Server 12.2.1.3  and 12.2.1.4.  We do not use Docker Store any longer please change to indicate to pull image from Oracle Container Registry container-registry.oracle.com/middleware/weblogic:<version>